client/src/components/BlockLinkField/tests/BlockLinkField-test.js

```diff
@@ -61,7 +61,6 @@ describe('BlockLinkField', () => {
         />
       );
 
-      expect(wrapper.instance().render()).toThrow();
```

Comment:
I remove this line because it seemed somewhat nonsensical in this context. If you actually expect a error to be thrown, you won't be able to reach the other lines below.